chef/cookbooks/barclamp/libraries/barclamp_library.rb

```diff
@@ -203,6 +215,31 @@ def usage
           self.owner
         end
 
+        def multipath?
+          self.class.multipath?(@device)
+        end
+
+        def held_by_multipath?
+          # we need to check if the holders of a device (if it has holders)
+          # are a multipath, for example:
```

Comment:
This is referring to "the holders of a device" so it should be plural no? May be my borked english :)